gctscript/examples/exchange/query_order.gct

```diff
@@ -2,7 +2,7 @@ fmt := import("fmt")
 exch := import("exchange")
 
 load := func() {
-   info := exch.orderquery("BTC Markets", "4491600698")
+   info := exch.orderquery("BTC Markets", "4491600698", currency.Pair{}, "spot")
```

Comment:
Thanks for making the changes! Last thing, I should have specified the line completely, sorry. Could you please update this line to be `info := exch.orderquery("binance", "4491600698", "BTC-USDT", "spot")` Just so it provides a complete example to anyone looking to use it
Then everything is good :)